weblate/trans/forms.py

```diff
@@ -633,10 +633,10 @@ class SimpleUploadForm(forms.Form):
         required=True,
     )
     fuzzy = forms.ChoiceField(
-        label=_("Processing of strings needing edit"),
+        label=_("Processing of \"Needs editing\" strings"),
         choices=(
             ("", _("Do not import")),
-            ("process", _("Import as string needing edit")),
+            ("process", _("Import as \"Needs editing\"")),
```

Comment:
```suggestion
            ("process", _("Import as such")),
```
What I would do, but possibly too complex?